apps/showcase/src/app/component-replacement/component-replacement.spec.ts

```diff
@@ -23,7 +26,8 @@ describe('ComponentReplacementComponent', () => {
     await TestBed.configureTestingModule({
       imports: [ComponentReplacementComponent,
         RouterModule.forRoot([]),
-        AsyncPipe]
+        AsyncPipe],
```

Comment:
tiny fix: new line between imports

```suggestion
      imports: [
        ComponentReplacementComponent,
        RouterModule.forRoot([]),
        AsyncPipe
      ],
```